### PR TITLE
Add support to run baremetal node disruptions

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit enables users to run disrupt baremetal using IPMI to be
able to understand the impact on OpenShift as well as applications
running on it.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
@paigerube14 @tsebastiani tested this on a baremetal cluster and it works as expected. Need your time when you get time please.

File: node-scenarios/Dockerfile [hunk 1]
```diff
@@ -4,6 +4,10 @@ FROM quay.io/krkn-chaos/krkn:latest
 
 ENV KUBECONFIG /home/krkn/.kube/config
 
+USER root
+RUN yum install -y OpenIPMI ipmitool
+USER krkn
```
Comment (paigerube14) [inline node-scenarios/Dockerfile:9]: do we need this user here?  Is that going to mess up any other commands?

Comment (chaitanyaenr) [reply node-scenarios/Dockerfile:9]: We basically use krkn user by default in our images. We have to switch to root just for the package install as krkn user doesn't have needed permissions and then switch back to krkn user. Thoughts?

**Review by paigerube14 — APPROVED** [review]
/lgtm

**Pull request merged**